From 00749fca9db235e916f94bf0521a5774bc4dc72f Mon Sep 17 00:00:00 2001
From: Kay Sievers <kay@vrfy.org>
Date: Sun, 4 Nov 2012 16:55:23 +0100
Subject: [PATCH 208/209] kmod-setup: mounting efivarfs, *after* we tried to
 mount it, is pointless

The mount() system call, which we issue before loading modules, will trigger
a modprobe by the kernel and block until it returns. Trying to load it again
later, will have exactly the same result as the first time.
(cherry picked from commit 3dfb265083347cb5700dc38f7cc0f479f378e6e9)
---
 src/core/kmod-setup.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/core/kmod-setup.c b/src/core/kmod-setup.c
index 383a6b2..20ab232 100644
--- a/src/core/kmod-setup.c
+++ b/src/core/kmod-setup.c
@@ -39,7 +39,6 @@ typedef struct Kmodule {
 static const KModule kmod_table[] = {
         { "autofs4",  "/sys/class/misc/autofs",    NULL } ,
         { "ipv6",     "/sys/module/ipv6",          NULL },
-        { "efivarfs", "/sys/firmware/efi/efivars", NULL },
         { "unix",     "/proc/net/unix",            NULL } ,
 };
 
-- 
1.8.1.5

